src/_data/defaultSchedule.yml

```diff
@@ -39,20 +39,28 @@ tuesday:
   - start: 13:00
     end: 14:00
     name: Lunch
-  - start: 16:00
-    end: 17:00
-    name: Interview practice
-  - start: 17:00
+  - name: Project intro
+    start: 17:00
+    end: 17:15
+    type: project
+    url: ../project
+  - name: Project set up, estimation and testing
```

Comment:
I'm not convinced this can all fit into half an hour. I remember recent cohorts easily spending half a day getting every ready (especially once they started using databases). Maybe this should just be "Repo setup & estimation", to imply they should be focusing on GH issues rather than code?